dev-util/cutter/cutter-1.3.recipe

```diff
@@ -84,15 +86,17 @@ BUILD()
 	mkdir -p build
 	cd build
 	export RADARE2_INC_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`/libr"
```

Comment:
Using `portPackageLinksDir` would be better, as it would allow building cutter even if the user has installed radare in the `/home/config` prefix

```diff
-	export RADARE2_INC_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`/libr"
+	export RADARE2_INC_DIR="$portPackageLinksDir/devel~libr2/.self/$relativeIncludeDir/libr"